README.md

```diff
@@ -30,79 +28,17 @@ This is a proof-of-concept application which generates a customized looking queu
 
 # Overview
 
-This web application provides a support queue system that could be embedded in E2C use cases and contains two views: Requester & Responder views. As you see in the figure above, there is a list of requests for a responder to manage or address to. In order for the responder to view any requests, they would need any requester to submit a request. Once a requester has submitted a request, responder will have few options to manage the request. Responder is able to skip or address the request by starting a session with the requester. This PoC is designed to address some of the use cases mentioned below:
+This web application offers a support queue system that can be integrated into E2C use cases. It empowers demo builders to craft a custom-branded experience, enabling users to dial into up to 4 queues, each tailored to provide a unique experience while awaiting connection to an agent. This Proof of Concept (PoC) is specifically designed to address the following use cases:
```

Comment:
This application generates customized web application links to demos that you create. In this context, a "demo" is a   page with custom background, company logo and Webex Call Queue(s) that make up the UI/controls of a the RoomOS device in PWA/Kiosk device. You can create multiple demos and activate them on a compatible device of your choice after logging-in. Once activated, the application uses a on-device Macro (that you need to also install) to make outbound direct call into Webex Call Queue on user action.

Reply:
It's not perfect - need to gather feedback from the team.